llvm/test/MC/ELF/rtti-proxy-gotpcrel.ll

```diff
@@ -1,4 +1,6 @@
-; RUN: llc %s -mtriple=x86_64-unknown-fuchsia  -o - | FileCheck %s
```

Comment:
Since there is no Fuchsia specific bits, I'd prefer `x86_64` (omitted components mean "unknown"), a generic ELF triple. This is clear that the behavior applies to all ELF based OSes (and discourages people from adding RUN lines for other OSes, which is unnecessary).